docs/core/diagnostics/runtime-interop-events.md

```diff
@@ -0,0 +1,32 @@
+
+# Interop Events
+
+These runtime events capture information about Microsoft intermediate language (MSIL) stub generation.  
+
+This category consists of the following event:
+
+- [ILStubGenerated](#ilstubgenerated-event)
+
+## ILStubGenerated Event
+
+|Keyword for raising the event|Level|  
+|-----------------------------------|-----------|  
+|`InteropKeyword` (0x2000)|Informational(4)|  
+  
+|Event|Event ID|Raised when|  
+|-----------|--------------|-----------------|  
+|`ILStubGenerated`|91|An IL Stub is generated.|
+
+|Field name|Data type|Description|  
+|----------------|---------------|-----------------|  
+|ModuleID|win:UInt16|The module identifier.|  
+|StubMethodID|win:UInt64|The stub method identifier.|  
+|StubFlags|win:UInt64|The flags for the stub:<br /><br /> `0x1` - Reverse interop.<br /><br /> `0x2` - COM interop.<br /><br /> `0x4` - Stub generated by NGen.exe.<br /><br /> `0x8` - Delegate.<br /><br /> `0x10` - Variable argument.<br /><br /> `0x20` - Unmanaged callee.<br /><br /> `0x40` - Struct Marshal|  
+|ManagedInteropMethodToken|win:UInt32|The token for the managed interop method.|  
+|ManagedInteropMethodNameSpace|win:UnicodeString|The namespace of the managed interop method.|  
+|ManagedInteropMethodName|win:UnicodeString|The name of the managed interop method.|  
+|ManagedInteropMethodSignature|win:UnicodeString|The signature of the managed interop method.|  
+|NativeMethodSignature|win:UnicodeString|The native method signature.|  
+|StubMethodSignature|win:UnicodeString|The stub method signature.|  
+|StubMethodILCode|win:UnicodeString|The MSIL code for the stub method.|  
```

Comment:
I believe the MSIL acronym has been deprecated in preference to "Common Intermediate Language" (CIL)

Reply:
If so I would fix throughout this review.